frankenphp.go

```diff
@@ -435,15 +435,17 @@ func go_register_variables(rh C.uintptr_t, trackVarsArray *C.zval) {
 	// FIXME: remove this debug statement
 	env[fmt.Sprintf("REQUEST_%d", rh)] = "1"
 
-	// TODO: batch this for performance
-	for k, v := range env {
-		ck := C.CString(k)
-		cv := C.CString(v)
+	le := len(env)
+	variables := make([]*C.char, le*2)
```

Comment:
This should probably be something like that:

```suggestion
	cArr := C.malloc(C.size_t(le) * C.size_t(unsafe.Sizeof(uintptr(0))))
	variables = unsafe.Slice(cArr, le)
	
	var i int
	for k, v := range env {
		variables[i] = C.CString(k)
		i++
		variables[i] = C.CString(v)
		i++
	}

        C.frankenphp_register_bulk_variables(cArr, C.int(le), trackVarsArray)

	for _, v := range variables {
		C.free(unsafe.Pointer(v))
	}
	
	C.free(cArr);
```